test/common/creation.go

```diff
@@ -31,21 +31,27 @@ var rbacAPIGroup = rbacv1.SchemeGroupVersion.Group
 var rbacAPIVersion = rbacv1.SchemeGroupVersion.Version
 
 // CreateChannelOrFail will create a Channel Resource in Eventing.
-// TODO(Fredy-Z): This is a workaround when there are both provisioner and Channel CRDs in this repo.
-//                isCRD needs to be deleted when the provisioner implementation is removed.
 func (client *Client) CreateChannelOrFail(name string, channelTypeMeta *metav1.TypeMeta, provisionerName string) {
```

Comment:
does this actually fails? do we have a default case for the switch that returns an error if that kind is invalid or something?

Reply:
Yes it will actually fail. Since I don't see any scenario that we need to handle the error explicitly instead of failing the test, all creation functions are provided in this format.

Reply:
but this will then panic due to a null pointer exception? I don't like panics much :)
 If we are failing in this manner in other places, I  think we should refactor and fail gracefully. It will make easier for reading the logs as well. 
I'm OK to leave it for now, but we should do some follow ups for proper error handling.